app/scripts/directives/columnnetworkvis.js

```diff
@@ -125,35 +128,21 @@ angular.module('globiProtoApp')
           var link = svg.selectAll('.link').data(links);
           var lineLinks = link.enter().insert('line')
             .on('mouseover', function() {
-              d3.select(this).style({'stroke-width': '4px', 'stroke' : '#2ED3DE'});
+              d3.select(this).classed('link-selected', true);
             })
             .on('mouseout', function() {
-              d3.select(this).style({'stroke-width': '2px', 'stroke' : '#999'});
+              d3.select(this).classed('link-selected', false);
             })
             .on('click', function(linkItem) {
               scope.$emit('linkClicked', linkItem);
             })
-            .attr('class', function(d) {
-              if (d.linkBack) {
-                // EXTREMELY important to still maintain link class because that's how the elements are selected
-                return 'link linkback';
-              } else {
-                return 'link';
-              }
-            })
+            .classed('link', true)
+            .classed('linkback', function(d) { return d.linkBack; })
```

Comment:
Does this also maintain the two classes, ie "link linkback" ?
Otherwise, when user goes "back" to an earlier node, the linkbacks will not be removed and you end up with arrows pointing to nowhere. This took me hours to figure out, so just want to make sure it still works, cause of course, we don't have unit tests ;-)